tools/ci-scripts/linux/image-builder/Dockerfile

```diff
@@ -49,5 +49,10 @@ RUN add-apt-repository ppa:cwchien/gradle && \
     gradle \
     openjdk-8-jdk-headless
 
+# Components for ghc
+RUN add-apt-repository ppa:hvr\ghc -y && \
+    apt-get update && \
+    apt-get -y install ghc-8.6.5
+
```

Comment:
@lalo is this an acceptable workaround for ghc installation issue during stack build?

Reply:
Let's give it a shot to unblock the rest of this PR, but give me some time.

Reply:
Looking at the logs of the [image build](https://msazure.visualstudio.com/One/_build/results?buildId=25186859), ghc seems to get installed in /opt. And it is not getting into the docker. Am not familiar enough with docker and the scripts to fix this mismatch.